include/psp2/kernel/processmgr.h

```diff
@@ -96,6 +99,29 @@ SceUInt32 sceKernelGetProcessTimeLow(void);
  */
 SceUInt64 sceKernelGetProcessTimeWide(void);
 
+SceUID sceKernelGetCurrentProcess(void);
+SceInt32 sceKernelGetRemoteProcessTime(SceUID processId, SceKernelSysClock *pClock);
```

Comment:
```suggestion
SceInt32 sceKernelGetRemoteProcessTime(SceUID processId, SceKernelSysClock *clock);
```
but it's ok, this file already has `pSysClock`